Makefile

```diff
@@ -101,6 +101,8 @@ services_image_registry=
 data_index_image_tag=
 jobs_service_image_tag=
 management_console_image_tag=
+runtime_application_image_namespace=
```

Comment:
I will be a PITA, but I would put the `registry` before the `namespace` as image's names are `registry/namespace/name:tag`

Reply:
At the moment, the rest of parameters follow this order: version, namespace, registry. Here, we don't have version, but the order matches with namespace, registry still.

Reply:
yeah right.
I modified the order as part of https://github.com/kiegroup/kogito-cloud-operator/pull/344/files.
So maybe could directly changed that here ?
or I will do it if that one is merged before

Reply:
ah ok, I missed some context here. Then, I will change the order also here. Thanks!